### DISCUSSION TITLE
Css not loading

### DISCUSSION THREAD

**Discussion by gevezex** [vueform/vueform#129] [Questions] [answered]
My issue is that the css is not loading (checked it with chrome inspect). So these were the steps I followed:

I created a new vue project with the vue cli:  `vue create test-vue-project`
 
For manual installation I selected these on the website:

Framework: Vue.js
Theme: Official

And created those config files what was suggested on the website and created a simple form with the vueform builder. 

When I start the dev env with `npm run serve` I get:
warn - The `content` option in your Tailwind CSS configuration is missing or empty.
warn - Configure your content sources or your generated CSS will be missing styles.

But it eventually starts. The problem is that my theme is not loading. Also I get this console warning:
main.js:9 Feature flag __VUE_PROD_HYDRATION_MISMATCH_DETAILS__ is not explicitly defined. You are running the esm-bundler build of Vue, which expects these compile-time feature flags to be globally injected via the bundler config in order to get better tree-shaking in the production bundle.

For more details, see https://link.vuejs.org/feature-flags.

Any clue what is happening?

**Comment by adamberecz** [comment]
On one hand you might need to create `tailwind.config.js` eg. by running: `npx tailwindcss init -p`.

On the other hand it seems like Vue CLI is not processing CSS included in `.js` files, so a workaround is to place Vueform's `.css` import to eg. you `App.vue`'s `<style>` block:

```vue
<!-- src/App.vue -->

<template>
  <div>
    <Vueform>
      <TextElement name="hello_world" label="Hello" placeholder="World" />
    </Vueform>
  </div>
</template> 
 
<style>
@import './../node_modules/@vueform/vueform/dist/vueform.css';
</style>

```

**Reply by gevezex** [reply, marked as answer]
I solved it by using vite. Now everything works. For whats worth it: 
```
npm create vite@latest my-vue-app -- --template vue
cd my-vue-app
npm install 
npm install @vueform/vueform
npm run dev
```

**Reply by adamberecz** [reply]
Yes, I wanted to recommend using Vite instead if possible, but I forgot. I'm glad you figured it out!
